.gitignore

```diff
@@ -9,6 +9,7 @@ dump.rdb
 
 # Distribution / packaging
 .Python
+.idea/
```

Comment:
If you use an IDE that leaves something in the project directory, please add it to your gitignore_global.